superset/migrations/versions/301362411006_add_execution_id_to_report_execution_.py

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add_execution_id_to_report_execution_log_model.py
+
+Revision ID: 301362411006
+Revises: 67da9ef1ef9c
+Create Date: 2021-03-23 05:23:15.641856
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '301362411006'
+down_revision = '67da9ef1ef9c'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column(
+        'report_execution_log',
+        sa.Column('execution_id', sa.VARCHAR(length=50), nullable=True)
```

Comment:
Can we use `sa.Column('uuid', UUIDType(binary=True))` just to be consistent with other entities in Superset?

https://github.com/apache/superset/blob/b34c86382525811fd707a9c133b9051d5c46a393/superset/migrations/versions/c501b7c653a3_add_missing_uuid_column.py#L69

Reply:
Yes! I wasn't aware there was a column type specifically for UUID's.

Reply:
Since we'll be having multiple log entries with the same UUID, would it be confusing to name the column `uuid`?

Reply:
I think we can call it UUID internally and just change the display name.